packages/indexer/src/indexer.test.ts

```diff
@@ -17,6 +16,8 @@ describe("Run Test", () => {
   async function cleanup() {
     try {
       await fs.unlink("file:memdb_indexer?mode=memory&cache=shared");
```

Comment:
Also, if vitest runs all tests in parallel it might cause issues if we use `file::memory:?cache=shared` everywhere. So i went ahead with named in memory db method.